code/src/components/PopularMoviesList.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import { Link } from 'react-router-dom'
+
+const PopularMoviesList = ({ movies }) => {
+    console.log('popularmovielist', movies)
+
+    return (
+        <section className='home-container'>
+            {movies.map((movie) => {
+                return (
+                    <section key={movie.id}>
+                        <Link to={`/details/${movie.id}`}>
+                            <div className='overlay-group'>
+                                <img src={`https://image.tmdb.org/t/p/w500${movie.poster_path}`} alt='poster' className='poster' />
+                                <div className='overlay-text'>
```

Comment:
There seems to be a padding on className overlay-text of 16 px making your movie cards rows uneven  and creating gaps. It's great that you made the background black. It makes it less obvious.